pkg/operator/staticpod/controller/staticpodfallback/static_pod_fallback_condition.go

```diff
@@ -103,11 +110,13 @@ func (fd *staticPodFallbackConditionController) sync(ctx context.Context, _ fact
 		}
 	}
 
+	// by default, the condition is in a non-degraded state
+	degradedCondition = degradedCondition.WithStatus(operatorv1.ConditionFalse)
```

Comment:
Would you mind setting this in an else clause (just to prevent someone from relying on this default value in the future)?